docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -74,6 +74,19 @@ relative to when data was written last to a rolled over index.
 The previous phase's actions must complete before {ILM} will check `min_age` and
 transition into the next phase.
 
+=== Phase Execution
+
+beta[]
+
+The current phase definition, of an index's policy being executed, is stored
+in the index's metadata. The phase and its actions are compiled into a series
+of discrete steps that are executed. Since some {ILM} actions are more complex
+and involve multiple actions against an index, each of these are done in their
+own step. This provides better visibility into the execution of the phases. The
```

Comment:
` This provides better visibility into the execution of the phases` -> I'm not sure about this sentence since this is not actually the reason we have steps (Though it is the reason why we expose steps in the ILM explain API). The reason we split the actions into steps is so we have better control over the execution of the lifecycle and so we know exactly which operations have been executed and which are still to be done.

Reply:
another sentence that is unnecessary